ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/OutputRecordsNode.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OutputRecordsNode {
```

Comment:
I thought about it but then the name of the array field would be the same for both input and output files and it would not be easy to know if a file is input or output if you just look at the content.